src/dr_llm/pool/pending_store.py

```
@@ -0,0 +1,459 @@
+                "sample_idx",
+                "payload_json",
+                "source_run_id",
+                "call_id",
+                "metadata_json",
+                "priority",
+                "status",
+                "worker_id",
+                "lease_expires_at",
+                "attempt_count",
+                "created_at",
+            ]
```

Ruff (0.15.7)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)